modules/open_data_schema_ckan/open_data_schema_ckan.module

```diff
@@ -6,77 +6,6 @@
  */
 
 /**
- * Implements hook_open_data_schema_map_endpoints_alter().
- */
-function open_data_schema_ckan_open_data_schema_map_endpoints_alter(&$records) {
-  $records[] = 'ckan_site_read';
-  $records[] = 'ckan_revision_list';
-  $records[] = 'ckan_package_revision_list';
-}
-
-/**
- * Implements hook_open_data_schema_map_load().
- */
-function open_data_schema_ckan_open_data_schema_map_load($machine_name) {
-  if ($machine_name == 'ckan_site_read') {
```

Comment:
These endpoints are now missing from the list of APIs, they are not defined in open_data_schema_map_dkan's features. See admin/config/services/odsm on the probo site vs http://dan.dkandemo.nuamsdev.com/admin/config/services/odsm